### PR TITLE
Check Visual Title #8

### LINKED ISSUE #7: Check Visual filter Component
### Explanation
It's in the top left. The buttons you will find are those of featured artists, hot artists and others. Its function in general is to classify the different types of artists that can be found on this web page, with which the user can have an ease when choosing an image of some artists in particular.

### Screenshots
image

### Checklist

- [ ] The Featured Artists button opens or searches for all the artists that are quite famous and leads the user to an image gallery.

- [ ] The trending artist button opens or searches for all the artists that are currently very fashionable and takes the user to an image gallery.

- [ ] The other artists button searches for any type of artist, whether famous, moderately famous, or not very famous. with it the page throws a gallery where they show the images of any artist.

- [ ] The buttons are in complementary color and code #F8F1F1 and their letters are in black with code #000000

### References

https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A51

### LINKED ISSUE #8: Check Visual Title
### Explanation

The title component is closely related to the filter, since the title reflects the option that was taken in the filter and returns a list of images depending on the option  chosen. In other words, the title is not partial and is subject to the filter exclusively here . In this case the color of the button is blue and its letters and words vary according to the filter.

### Screenshots
image

### Checklist

- [ ] The titles change according to the option that is chosen in filter.
- [ ] The color of the letter in the title does not change, it remains stable.
- [ ] The images obey the inscribed title, that is, they are in accordance with the chosen option in filter

### References
https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A51

### LINKED ISSUE #14: Review Optional Slider
### Explanation
Allow to change button colors, change title, remove title or change background.

 ### Screenshots
image

### Step by step guide to reproduce the issue

- Open the app https://art-marketplace.pages.dev/
- Go to the products page in https://art-marketplace.pages.dev/products/
- Click on "Categorías" button at sidebar
- Click on any item of the list

### Checklist

- [ ] The arrowheads are used to go through the categories or elements
- [ ] The arrow circumferences have a complementary color with code #F8F1F1
- [ ]  Arrowheads are tertiary color coded #11698E
- [ ] The category buttons have a rectangle style and their tips or vertices are somewhat rounded
- [ ] The categories or elements have various names, including minimalist, realistic and many more with code letters #000000

### References
https://www.figma.com/file/zzEwuPfGTiZh4EA11MyZ5c/Sencico?node-id=65%3A51

### PR DESCRIPTION
**Pull request by Eghinner**
fixes #7
screenshot4
fixes #8 
screenshot5
fixes #14 
screenshot1
screenshot2
screenshot3

### CODE REVIEW

**Comment by Eghinner** [comment]
I just realized that home slider also changed, i'll solve it

**Comment by e-azocar** [comment]
This changes is very good, but I think that this issue was just change a simple title.
Really good job

File: src/components/Slider.js [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+        }}
       onClick={onClick}
-    />
+    >
+    <svg style={{
+      position: "absolute",
+      transform: "translate(-50%, -50%)"
+      }} xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+      <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={3} d="M15 19l-7-7 7-7" />
+    </svg>
```
Comment (Israel-Laguan) [inline src/components/Slider.js]: Please move this svg to a component

File: src/components/Slider.js [hunk 2]
```diff
@@ -1,92 +1,137 @@
...
+            </div>
+          </div>
+        ))}
+      </Slider>
+    </div>
+    );
 };
 
-export default ProductSlider;
+export default ProductSlider;
```
Comment (Israel-Laguan) [inline src/components/Slider.js]:
```suggestion
export default ProductSlider;

```

**Review by Israel-Laguan — APPROVED** [review]
@e-azocar @Katsu08 please check visually

File: src/components/Slider.js (hunk 2 above)
Comment (Israel-Laguan) [reply src/components/Slider.js]: To avoid EOF issue

**Review by e-azocar — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Very good! Thanks for the changes!

**Pull request merged**